lib/color_guard.rb

```diff
@@ -29,6 +29,10 @@ def deactivate!(feature_name)
       end
     end
 
+    def all
+      store.all
+    end
```

Comment:
Yeah I was contemplating about that.

The same could be said for `#find`, "find what?"

I was trying to stick with it more Rails-like db calls.